04-mpu6050/script/mpu6050.sh

```diff
@@ -6,3 +6,17 @@ then
 fi
 
 i2cset -y 0 0x68 0x6b 0
+
+while [ true ]
+do
+	tempHexH=`i2cget -y 0 0x68 0x41`
+	tempHexL=`i2cget -y 0 0x68 0x42`
+
+	temperature=$(( -(((tempHexH << 8) + tempHexL) ^ ((1 << 0x10) - 1)) ))
```

Comment:
This conversation is not correct for positive values.

Reply:
Can you, please, explain why? It seems, like temperature values that script outputs are pretty close to reality

Reply:
Please consider the positive temperature values (Above 36.5 degrees in Celsius scale) Using the purposed conversation will give huge negative numbers instead of 37 degrees and above.

Reply:
Got it. Thank you!

Reply:
Is the problem solved?